client/my-sites/stats/stats-chart-tabs/utility.js

```diff
@@ -194,6 +194,10 @@ function addTooltipData( chartTab, item, period, customRange = {} ) {
 					className: 'is-visitors',
 					icon: <Icon className="gridicon" icon={ people } />,
 				} );
+			}
+
+			// Ensure the Views/Visitors ratio number is finite.
+			if ( item.data.visitors > 0 ) {
```

Comment:
Looks like the comment doesn't match what we're doing. If `item.data.visitors` is `Infinity`, `Infinity > 0` will be `true`, which is the opposite of what the comment claims. Did you mean to do:

```
if( Number.isFinite( item.data.visitors ) )
```

Reply:
I see this same no-infinity check is done above - perhaps we could just reuse that same condition and put the item in there?

Reply:
If `item.data.visitors` is `Infinity`, the condition `item.data.visitors > 0` is `true` and the **Views/Visitors ratio** should be `0` unless the `item.data.views` is also `Infinity`. So, I think the case would conform to the comment.

However, my point was to remove the case of `0` visitors as the division would be invalid when the denominator is `0`.

> I see this same no-infinity check is done above

Do you mean [the line](https://github.com/Automattic/wp-calypso/blob/0d3220a2db3d61989b5020890e8fa0f02418f9e9/client/my-sites/stats/stats-chart-tabs/utility.js#L190) where the `item.data.views / item.data.visitors` calculation was?

Reply:
I see what you mean, but this means this is still quite confusing.

If we want to ensure the Views/ Visitors ratio number is finite, can't we express this through code? Like, literally checking if the ratio is finite. Then the comment won't actually be necessary, and the code will be more readable.

Reply:
> checking if the ratio is finite. Then the comment won't actually be necessary

Yes, that was one of my options, which would be more readable. Thanks for the feedback!

Reply:
Followed up in https://github.com/Automattic/wp-calypso/pull/101208.